JASP-Desktop/components/JASP/Widgets/FileMenu/PrefsAdvanced.qml

```diff
@@ -206,10 +206,45 @@ ScrollView
 				checked:			preferencesModel.generateMarkdown
 				onCheckedChanged:	preferencesModel.generateMarkdown = checked
 				toolTip:			qsTr("Enabling this will generate markdown helpfile from the info at qml options.")
+				KeyNavigation.tab:	checkForLC_CTYPE_C
+				KeyNavigation.down:	checkForLC_CTYPE_C
+
+			}
+		}
+		
+		PrefsGroupRect
+		{
+			id:			windowsSpecific
+			visible:	WINDOWS
+			enabled:	WINDOWS
+			title:		qsTr("Windows Workarounds")
+			
+			CheckBox
```

Comment:
I thought the checkbox behaviour was a bit strange.
One checkbox appears when you click the other, but they cannot both be on at the same time.
This sound like a radiobutton job to me:
```
LC_CTYPE:
(x) Auto
( ) Set to "C"
( ) Disabled
```

Reply:
It also crashes JASP amazingly easy. I tried to interact with it in a regular fashion, but it is super sensitive to crashing. Should there be a little delay before JASP automatically does something with this value?

Reply:
Yeah maybe it is better to make it a radiobutton actually, I just didn't want people to get confused.
Also, I really don't care if it crashes JASP (also, it didn't do that for me, but I guess I have a supercomputer or something?)

Like, people really shouldn't be changing this anyway.

Reply:
And no more timers...